feature-libs/organization/account-summary/components/details/header/account-summary-header.component.ts

```diff
@@ -0,0 +1,121 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Address, LanguageService, TranslationService } from '@spartacus/core';
+import {
+  AccountSummaryDetails,
+  AccountSummaryFacade,
+} from '@spartacus/organization/account-summary/root';
+import { Card } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+import { map, switchMap, take } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-account-summary-header',
+  templateUrl: './account-summary-header.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AccountSummaryHeaderComponent {
+  notApplicable: string;
+  headerDetails$: Observable<AccountSummaryDetails> = this.languageService
+    .getActive()
+    .pipe(switchMap(() => this.accountSummaryFacade.getAccountSummary()));
+
+  constructor(
+    protected accountSummaryFacade: AccountSummaryFacade,
+    protected translation: TranslationService,
+    private languageService: LanguageService
+  ) {
+    this.translation
+      .translate('orgAccountSummary.details.notApplicable')
+      .pipe(take(1))
+      .subscribe((text) => (this.notApplicable = text));
```

Comment:
if `take(1)`, `notApplicable` will not be changed if changing language. Maybe remove `take(1)` and unsubscribe it in `ngOnDestroy`. It is better to use `combineLatest([this.translation.translate('orgAccountSummary.details.uid'),     this.translation.translate('orgAccountSummary.details.notApplicable')])...`

Reply:
removed take(1) and unsubscribed in ngOnDestroy. I did no use combineLatest because the notApplicable is used in multiple places.